packages/react/src/patterns/blocks/ContentBlockMedia/README.md

```diff
@@ -63,11 +63,11 @@ function App() {
     type: 'local'|'jump'|'external'
     heading: 'Feature Link heading',
     card: {
-      href: 'https://ibm.com',
+      href: 'https://www.example.com',
       title: 'Consectetur adipisicing elit',
       image: {
-        defaultImage: 'https://picsum.photos/id/2/672/672',
-        alt: 'featured link image',
+        defaultImage: 'https://dummyimage.com/672x672',
```

Comment:
Should this have an aspect ratio and new styling?

Reply:
I didn't bother to put that in this part of the README, as I'd have to document the service's API. If you think it should be in the docs we can add a few lines for usage guidance.